app/App.js

```diff
@@ -51,41 +57,46 @@ export default class App extends Component {
       return this.startPulsing(animations.knownSite);
     }
     if (!euCountries.includes(countryCode) && !localStorage[tokenKey]) {
-      return this.startPulsing(animations.euTraffic);
+      return this.startPulsing(animations.companySettings);
     }
-    if (localStorage[tokenKey] && Xcoobee.config.campaignReference) {
-      return this.startPulsing(animations.defaultOptions);
+    if (localStorage[tokenKey] && XcooBee.kit.config.campaignReference) {
+      return this.startPulsing(animations.userSettings);
     }
     return animations.noAnimation;
   }
 
   checkRequiredFields() {
     requiredFields.forEach((field) => {
-      if (!Xcoobee.config[field]) {
+      if (!XcooBee.kit.config[field]) {
         this.errors = true;
         console.error(`${field} field is required as initialization parameter`);
       }
     });
   }
 
   fetchCrowdAI() {
-    const { campaignName, campaignReference } = Xcoobee.config;
+    const { campaignReference } = XcooBee.kit.config;
 
-    if (campaignName && campaignReference && localStorage[tokenKey]) {
+    if (campaignReference && localStorage[tokenKey]) {
       const query = `query CrowdRating($campaign_name: String!) {
         crowd_rating(campaign_name: $campaign_name) {
           cookie_type value
         }
       }`;
 
-      graphQLRequest(query, { campaign_name: campaignName }, localStorage[tokenKey])
+      graphQLRequest(query, { campaign_name: window.location.host }, localStorage[tokenKey])
         .then((res) => {
-          const { campaign } = this.state;
-
-          campaign.cookies.forEach((cookie) => {
-            cookie.checked = res.crowd_rating.find(item => item.cookie_type === cookie.type).value >= 0.8;
-          });
-          this.fetchLocation(true);
+          if (res && res.crowd_rating) {
+            const { config } = XcooBee.kit;
+
+            config.cookies.forEach((cookie) => {
+              cookie.checked = res.crowd_rating.find(item => item.cookie_type === cookie.type).value >= 0.8;
+            });
+            this.fetchLocation(true);
+          } else {
+            console.error("Wrong campaign name.");
```

Comment:
Your error messages especially to console should be prefixed with a constant string like [XcooBee:kit]
```suggestion
            console.error("[XcooBee:kit] Wrong campaign name.");
```